src/style.css

```diff
@@ -3,9 +3,15 @@
   padding: 0;
 }
 
+body {
+  background-image: none;
+}
+
 .all-sections {
   display: flex;
   justify-content: space-around;
+  flex-wrap: wrap;
+  gap: 50px;
 }
 
 .head-and-button {
```

Comment:
- [ ] Your app styling looks rather generic. Kindly improve your styling based on the requirements

  ![fury](https://user-images.githubusercontent.com/81468675/198260065-84673623-b0d1-44fb-8c3e-8cb49c4c0624.jpg)

Reply:
@julie-ify Apologies, I mistakenly never ran "npm run build" into the styling branch. But it has now been corrected.